ipv8/test/REST/rest_base.py

```diff
@@ -61,7 +61,7 @@ async def start_api(self):
         await self.rest_manager.start(0)
         self.rest_port = self.rest_manager.site._server.sockets[0].getsockname()[1]
 
-    async def unload(self):
+    async def stop(self, **_):
```

Comment:
Please remove the `**_` instances.

Reply:
The original `stop` method does have a `stop_loop` keyword argument:

```
async def stop(self, stop_loop=True):
```

Therefore, if I would use the `MockIPv8` in a Tribler session and implement the requested change, it would be called with this keyword argument, and failing.

Reply:
That is a fair use case, but then you are breaking with the function signature. Please use `stop_loop` instead of `_` then.

Reply:
Actually, if this is used as an `IPv8` drop-in replacement, it should really be a `IPv8` subclass.. but that's for another PR.

Reply:
Implemented, at the cost of two pylint errors.